qt/aqt/editor.py

```diff
@@ -91,13 +98,23 @@ class Editor:
     """
 
     def __init__(
-        self, mw: AnkiQt, widget: QWidget, parentWindow: QWidget, addMode: bool = False
+        self,
+        mw: AnkiQt,
+        widget: QWidget,
+        parentWindow: QWidget,
+        addMode: bool | None = None,
+        *,
+        editorMode: EditorMode = EditorMode.EDIT_CURRENT,
     ) -> None:
         self.mw = mw
         self.widget = widget
         self.parentWindow = parentWindow
         self.note: Note | None = None
-        self.addMode = addMode
+        # legacy argument provided?
+        if addMode is not None:
+            editorMode = EditorMode.ADD_CARDS if addMode else EditorMode.EDIT_CURRENT
+        self.addMode = editorMode is EditorMode.ADD_CARDS
+        self.editorMode = editorMode
```

Comment:
Didn't notice this before merging, but ideally this would be snake_case, not camelCase. We can't fix our old code at the moment, but should probably try to use Python convention in the new code we introduce, when we're not referencing the Qt API.